readme.md

```diff
@@ -42,3 +42,14 @@ should be ready to go.
 
 **It is CRITICAL that whole `app/`, `log/` and `temp/` directories are not accessible directly
 via a web browser. See [security warning](https://nette.org/security-warning).**
+
+Notice: Composer PHP version
+----------------------------
+This project forces `PHP 5.6` as your PHP version for Composer packages. If you have different version on production you should change it in `composer.json`.
+```json
```

Comment:
Yes, I know. This is better for users with old php. I know that minimum php version for current web-project is php 5.6.